### PR TITLE
Feature login component

### PR DESCRIPTION
**Pull request by liannivin**
- [x] It contains Login component.
- [x] It must use semantic HTML
- [x] It must use React
- [x] It must be Responsive
- [x] It must use CSS in JS
- [x] It must have tests
- [x] It must use Reach Router
- [x] It could use Reach UI components
- [x] It must pass with green the Accessibility tests of the Chrome Audits
- [x] It must pass with green the Best Practices tests of the Chrome Audits
- [x] It must pass with green the SEO tests of the Chrome Audits

### CODE REVIEW

File: src/views/login.js [hunk 1]
```diff
@@ -1,9 +1,125 @@
...
+            width: "50%",
+            maxWidth: 360
+          }}
+          onSubmit={handleSubmit}
+        >
+          <h1 css={{ margin: 0, fontWeight: "500", textAlign: "center" }}>
+            Welcome to Chaos
+          </h1>
+          <div css={{ margin: "2.5rem 1rem" }}>
+            <input
```
Comment (condef5) [inline src/views/login.js]: Hey bro you could change the inputs by the los `ui.js` file?

Comment (AngieCristina) [reply src/views/login.js]: :)

**Comment by condef5** [comment]
Good job bro, don't forget the accessibility :)

File: src/components/navbar.js [hunk 2]
```diff
@@ -3,7 +3,12 @@ import React from "react";
 import { jsx } from "@emotion/core";
 import { Link } from "@reach/router";
 
-function Navbar() {
+function Navbar({ user }) {
+  function deleteSession() {
```
Comment (condef5) [inline src/components/navbar.js]: I think you have to test this component.

**Pull request merged**